org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/gui/custom/JSelectTrackModel.java

```diff
@@ -140,14 +141,29 @@ public void setPrimaryTrack(final TrackWrapper newPrimary)
   }
 
   @Override
-  public void setTracks(final List<TrackWrapper> tracks)
+  public boolean setTracks(final List<TrackWrapper> tracks)
```

Comment:
Aaah @saulhidalgoaular - could you indicate the meaning of the returned value here, too.  Well, if it's not documented in the parent class.

Reply:
Good morning. Yes, javadoc has been added to the interface. I am adding it here too to make the code more readable.